### DISCUSSION TITLE
Linux version

### DISCUSSION THREAD

**Discussion by LiamM32** [Open-Systems-Pharmacology/Forum#1888] [Feature Requests & Ideas]
Make PK-Sim run on Linux.

**Comment by Yuri05** [comment]
In fact, it seems that PK-Sim runs under Linux via **Wine**. E.g. to run PK-Sim under Ubuntu:
* **manually install the latest Wine version** as described under https://www.omgubuntu.co.uk/2023/01/install-wine-on-ubuntu 
* install **winetricks** 
* install .NET 4.8, missing fonts and Win10 libs via `winetricks -q dotnet48 corefonts renderer=vulkan win10`
* Download and unzip the latest PK-Sim portable from https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/latest
* Switch to the PK-Sim folder and start PK-Sim via `wine ./PKSim.exe`

**Reply by rwmcintosh** [reply]
Can it work on MacOS via Wine also?

**Reply by msevestre** [reply]
Wine can make everything better

**Reply by rwmcintosh** [reply]
Works for MacOS too. Fonts are a bit blech... as is the smoothing.
![image](https://github.com/user-attachments/assets/b39c8908-8efc-4fec-a046-97ffbba4fc32)
